src/test/java/com/puppycrawl/tools/checkstyle/PackageObjectFactoryTest.java

```diff
@@ -57,7 +57,7 @@ public void testCtorException() {
             fail("Exception is expected");
         }
         catch (IllegalArgumentException ex) {
-            assertEquals("moduleClassLoader must not be null", ex.getMessage());
+            Assert.assertEquals("moduleClassLoader must not be null", ex.getMessage());
```

Comment:
@romani To follow the rest of the pattern, `Assert.assertEquals` should be a static import.